src/components/Documents/ConceptClusterDialog.vue

```diff
@@ -18,10 +18,21 @@
             </p>
             <p class="text-subtitle1">
               <b>{{ t('status') }}:</b> {{ graphPipelineStatus }}
-              <q-spinner v-if="graphPipelineInterval" size="xs" class="q-ml-sm" />
-              <q-icon v-else-if="isGraphPipelineFailed" name="bolt" color="red" />
-              <q-icon v-else-if="isGraphPipelineFinished" name="check" color="positive" />
+                <q-spinner v-if="graphPipelineInterval" size="xs" class="q-ml-sm" />
+                <q-icon v-else-if="isGraphPipelineFailed" size="xs" name="help" color="red">
+                  <q-tooltip :delay="250">
+                    {{ graphPipelineStatusDetails }}
+                  </q-tooltip>
+                </q-icon>
```

Comment:
We don't use `q-tooltip` anywhere else except for the code search field. I'd suggest to replace these occurrences with a simple `title` attribute, to be more consistent.

```suggestion
                <q-icon v-else-if="isGraphPipelineFailed" size="xs" name="help" color="red" :title="graphPipelineStatusDetails" />
```

Reply:
I did it here. Don't know if you're suggesting of changing it for the buttons as well, but I would not know how to do it there with the conditionals.

Reply:
The best way would be to introduce a computed value that contains the if statements. This would also make the HTML template a bit clearer.